File: src/main/java/santa/seedcopy/SeedCopyMod.java
package santa.seedcopy;

import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
import net.minecraftforge.fml.common.event.FMLServerStartingEvent;

@Mod(name = SeedCopyMod.NAME, modid = SeedCopyMod.MODID, version = SeedCopyMod.VERSION)
public class SeedCopyMod {
    public static final String NAME = "SeedCopy";
    public static final String MODID = "seedcopy";
    public static final String VERSION = "3.0.0";

    @Mod.EventHandler
    public void preInit(FMLPreInitializationEvent event) {
        Config.load(event);
    }

    @Mod.EventHandler
    public void onServerStart(FMLServerStartingEvent event) {
        event.registerServerCommand(new SeedCopyCommand());
        SeedCopyLogger.printLogMessage("SeedCopy command has been registered.");
    }
}

File: src/main/kotlin/santa/seedcopy/SeedCopyMod.kt
package santa.seedcopy

import net.minecraftforge.fml.common.Mod
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent
import net.minecraftforge.fml.common.event.FMLServerStartingEvent

@Mod(
  modid = SeedCopyMod.MOD_ID,
  name = SeedCopyMod.NAME,
  version = SeedCopyMod.VERSION,
  modLanguage = "kotlin",
  modLanguageAdapter = "io.drakon.forge.kotlin.KotlinAdapter")
object SeedCopyMod {
    const val NAME = "SeedCopy"
    const val MOD_ID = "seedcopy"
    const val VERSION = "3.0.0"

    @Mod.EventHandler
    fun preInit(event: FMLPreInitializationEvent) {
        Config.load(event)
    }

    @Mod.EventHandler
    fun onServerStart(event: FMLServerStartingEvent) {
        event.registerServerCommand(SeedCopyCommand())
        SeedCopyLogger.printLogMessage("SeedCopy command has been registered.")
    }
}